dsl/procedures/importMicroservices/steps/importMicroservices.groovy

```diff
@@ -45,22 +45,28 @@ if (environmentProjectName && environmentName && clusterName) {
 // write out the docker compose to the workspace directory
 String dir = System.getenv('COMMANDER_WORKSPACE')
 File composeFile = new File(dir, 'docker-compose.yml')
-composeFile << dockerComposeContent
+String composeContent = dockerComposeContent
+composeFile << composeContent.replace('#','').replace("\t", "")
```

Comment:
'#' is used for comments in Docker Compose.  Why are we removing '#' from the file?